cashier-sample-google-play-billing/build.gradle

```diff
@@ -24,7 +24,7 @@ dependencies {
     implementation 'com.getkeepsafe.cashier:cashier-google-play-billing:0.3.5'
     implementation 'com.getkeepsafe.cashier:cashier-google-play-billing-debug:0.3.5'
 
-    implementation 'com.android.support:recyclerview-v7:28.0.0'
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
```

Comment:
Nit: This could be added to `deps` map

Reply:
I didn't move it to `deps` deliberately - I want to keep samples as separate codebase and use `deps` and other shared gradle files in the library codebase only to keep it clean.